server/src/main/java/org/elasticsearch/transport/SniffConnectionStrategy.java

```diff
@@ -490,6 +491,11 @@ public String toString() {
     }
 
     private static DiscoveryNode resolveSeedNode(String clusterAlias, String address, String proxyAddress) {
+        var seedVersion = new DiscoveryNode.VersionInformation(
```

Comment:
If this is used a few times (ie version info based on current), maybe it should be a constant in DiscoveryNode or VersionInformation